src/BulletPlugin/BulletCollisionDetector.cpp

```diff
@@ -586,8 +586,8 @@ void BulletCollisionDetectorImpl::detectCollisions(boost::function<void(const Co
 void BulletCollisionDetectorImpl::detectObjectCollisions(btCollisionObject* object1, btCollisionObject* object2, CollisionPair& collisionPair)
 {
     vector<Collision>& collisions = collisionPair.collisions;
-    btCollisionObjectWrapper objectWrapper1(0, object1->getCollisionShape(), object1, object1->getWorldTransform(), -1, -1);
-    btCollisionObjectWrapper objectWrapper2(0, object2->getCollisionShape(), object2, object2->getWorldTransform(), -1, -1);
+    btCollisionObjectWrapper objectWrapper1(0, object1->getCollisionShape(), object1, object1->getWorldTransform());
+    btCollisionObjectWrapper objectWrapper2(0, object2->getCollisionShape(), object2, object2->getWorldTransform());
```

Comment:
Note: this change is wrong for Bullet >= 2.82 (cf. https://github.com/bulletphysics/bullet3/commit/9eddd42347b4d766fabaaff9f72d69632cb82f48).